### DISCUSSION TITLE
Xbox

### DISCUSSION THREAD

**Discussion by FoxyEnzo** [obsproject/obs-studio#5689] [General]
Quando vai lançar pra xbox to querendo muito

**Comment by feetstv** [comment]
It will launch on Xbox when Xbox runs the standard desktop version of Windows. :-)

So it won't happen.

You can broadcast to Twitch if you install the Twitch app on your Xbox.

**Comment by WizardCM** [comment]
We have no plans on releasing an Xbox version of OBS Studio.

**Comment by Gattsaki** [comment]
Me too, But, The OBS need one Update to work properly in the  console.
